doc/config.example.yaml

```diff
@@ -368,6 +369,26 @@ registry:
 
   # Zone responsible for flash nodes registration
   flash:
-    # Having separate authoritative server increases redundancy and reduces load,
+    # Having one or more secondary servers increase redundancy and reduces load,
     # hence this configuration, while unlikely in a small to medium network, is supported.
+    authoritative: true
+
+    # Redirect server for REST endpoints of the registry
+    redirect_primary: http://nsHidden.bosagora.io
+
+    # List of servers to AXFR zone transfer from
+    #
+    # Servers are tried in the configuration order until zone transfer can be
+    # completed from a server.
+    primary_servers:
+      - ns1.bosagora.io
+
+    # A secondary server can be primary of an another server, thus secondary
```

Comment:
No it can't. It can be an "upstream" server, but primary == ultimate authority on the zone.